docs/UserGuide.md

```diff
@@ -146,6 +150,16 @@ Format: `present INDEX date/dd-MM-yyyy [pu/HH:mm do/HH:mm]`
 * Pick-up and drop-off times are either specified **together** as a pair, or **not at all**.
 * Pick-up time must be **before** the drop-off time.
 
+<div markdown="block" class="alert alert-info">
+
+**:information_source: About `present` command:**<br>
+
+This command is designed to enable you to mark the attendance of pets for **any** date. If you have forgotten to mark
+the attendance a few weeks back, you can do so. If you want to mark the attendance of a pet for future dates because
```

Comment:
"If you want" can be changed to "If you wish" to standardise with the other commands.